src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -121,7 +121,7 @@ export default class PropertyAccessorsInstrumentation extends SandboxBase {
             },
 
             autocomplete: {
-                condition: domUtils.isInputElement,
+                condition: input => domUtils.isInputElement(input) && domUtils.isDomElement(input),
```

Comment:
I think we should add `isDomElement` everywhere - `isInputElement`, `isHtmlElement`, `isBodyElement` etc.